join-chat-to-teams-meeting/Chat Experience with Inline Image/package.json

```diff
@@ -4,21 +4,21 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "start": "webpack-dev-server --open --config webpack.config.js",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@azure/communication-calling": "^1.17.1",
-    "@azure/communication-chat": "1.4.0-beta.2",
+    "@azure/communication-calling": "^1.19.1",
+    "@azure/communication-chat": "1.4.0",
     "@azure/communication-common": "^2.3.0",
-    "@azure/communication-identity": "^1.2.0",
-    "@azure/communication-signaling": "1.0.0-beta.21"
+    "@azure/communication-identity": "^1.3.0"
   },
   "devDependencies": {
-    "webpack": "^4.42.0",
-    "webpack-cli": "^3.3.11",
-    "webpack-dev-server": "^3.10.3"
+    "webpack": "^5.89.0",
+    "webpack-cli": "^5.1.4",
+    "webpack-dev-server": "^4.15.1"
```

Comment:
are we going to update web pack for file sharing?

Reply:
Not here; that needs to be done as part of the updates with the file sharing PR #180